SPECS/nodejs/CVE-2022-25883.patch

```diff
@@ -1,326 +0,0 @@
-From 12b07816ba23e72527d8c36108889d7ee7cbfe4f Mon Sep 17 00:00:00 2001
```

Comment:
Patch no longer applies as https://nvd.nist.gov/vuln/detail/CVE-2022-25883 affect semver versions up to 7.5.2, but nodejs 20.10.0 uses semver 7.5.4 https://github.com/nodejs/node/blob/v20.10.0/deps/npm/node_modules/semver/package.json#L3